### ISSUE TITLE
Crash in 1.3.1

### ISSUE THREAD

**Issue by farwayer** [kmagiera/react-native-reanimated#448]
React Native 0.60.3

```js
Invariant Violation: [5,{"type":"value","value":"<<NaN>>"}] is not usable as a native method argument
```
```js
invariant (browser.js:45)
enqueueNativeCall (MessageQueue.js:318)
fn (NativeModules.js:136)
__nativeInitialize (AnimatedNode.js:118)
__attach (AnimatedNode.js:63)
AnimatedParam (AnimatedParam.js:9)
createAnimatedParam (AnimatedParam.js:32)
createAnimatedFunction (AnimatedFunction.js:29)
(anonymous) (Card.tsx:132)
loadModuleImplementation (require.js:321)
guardedLoadModule (require.js:210)
metroRequire (require.js:128)
(anonymous) (StackItem.tsx:6)
loadModuleImplementation (require.js:321)
guardedLoadModule (require.js:210)
metroRequire (require.js:128)
(anonymous) (Stack.tsx:15)
loadModuleImplementation (require.js:321)
guardedLoadModule (require.js:210)
metroRequire (require.js:128)
(anonymous) (StackView.tsx:4)
loadModuleImplementation (require.js:321)
guardedLoadModule (require.js:210)
metroRequire (require.js:128)
(anonymous) (stack.js:2)
loadModuleImplementation (require.js:321)
guardedLoadModule (require.js:210)
metroRequire (require.js:128)
(anonymous) (index.js:1)
loadModuleImplementation (require.js:321)
guardedLoadModule (require.js:210)
metroRequire (require.js:128)
(anonymous) (router.js:3)
loadModuleImplementation (require.js:321)
guardedLoadModule (require.js:210)
metroRequire (require.js:128)
(anonymous) (index.js:3)
loadModuleImplementation (require.js:321)
guardedLoadModule (require.js:210)
metroRequire (require.js:128)
(anonymous) (index.js:4)
loadModuleImplementation (require.js:321)
guardedLoadModule (require.js:210)
metroRequire (require.js:128)
(anonymous) (index.js:7)
loadModuleImplementation (require.js:321)
guardedLoadModule (require.js:210)
metroRequire (require.js:128)
(anonymous) (index.js:1)
loadModuleImplementation (require.js:321)
guardedLoadModule (require.js:201)
metroRequire (require.js:128)
(anonymous) (index.bundle?platform=android&dev=true&minify=false:169695)
executeApplicationScript (RNDebuggerWorker.js:2)
(anonymous) (RNDebuggerWorker.js:2)
Worker.postMessage (async)
Gn.e.onmessage (bundle.js:10)
await in Gn.e.onmessage (async)
Gn.e.onmessage (bundle.js:10)
```

**Comment by osdnk** [comment]
Please use master for now

**Comment by osdnk** [comment]
Released 1.3.2
